pallets/gear/src/benchmarking/mod.rs

```diff
@@ -1840,6 +1840,36 @@ benchmarks! {
         sbox.invoke();
     }
 
+    instr_call_indirect_per_result {
+        let r in 0 .. T::Schedule::get().limits.results;
+
+        #[cfg(not(force_wasmer_cranelift_i_know_what_i_do))]
+        let _ = r;
+
+        #[cfg(force_wasmer_cranelift_i_know_what_i_do)]
+        let mut sbox = {
+            let num_elements = T::Schedule::get().limits.table_size;
+            Sandbox::from(&WasmModule::<T>::from(ModuleDefinition {
+                aux_body: Some(body::repeated_dyn(r, vec![RandomI64(0, num_elements as i64)])),
+                aux_res: vec![ValueType::I64; r as usize],
+                handle_body: Some(body::repeated_dyn(INSTR_BENCHMARK_BATCH_SIZE, vec![
+                    RandomI32(0, num_elements as i32),
+                    Regular(Instruction::CallIndirect(r.min(1), 0)), // aux signature: 1 or 0
+                    DropRepeated(r as usize),
+                ])),
+                table: Some(TableSegment {
+                    num_elements,
+                    function_index: OFFSET_AUX,
+                    init_elements: Default::default(),
+                }),
+                .. Default::default()
+            }))
+        };
+    }: {
+        #[cfg(force_wasmer_cranelift_i_know_what_i_do)]
+        sbox.invoke();
```

Comment:
I think this leads to the fact that automatic benchmarks run would zeroes this weight in weights.rs